editor/editor_file_system.cpp

```diff
@@ -119,6 +119,11 @@ bool EditorFileSystemDirectory::get_file_import_is_valid(int p_idx) const {
 	return files[p_idx]->import_valid;
 }
 
+uint64_t EditorFileSystemDirectory::get_file_modified_time(int p_idx) const {
+	ERR_FAIL_INDEX_V(p_idx, files.size(), false);
```

Comment:
Hey compiler, mind throwing a warning here?
You're returning `false` in a method with type `uint64_t`.

Reply:
Ah, good catch. Thanks!